pkg/secrethub/service.go

```diff
@@ -54,25 +54,19 @@ func (s serviceService) Create(path string, description string, credentialCreato
 		return nil, errio.Error(err)
 	}
 
-	credentialRequest, err := s.client.createCredentialRequest(credentialCreator.Verifier(), credentialCreator.Metadata())
+	credentialRequest, err := s.client.createCredentialRequest(credentialCreator.Encrypter(), accountKey, credentialCreator.Verifier(), credentialCreator.Metadata())
 	if err != nil {
 		return nil, errio.Error(err)
 	}
 
-	accountKeyRequest, err := s.client.createAccountKeyRequest(credentialCreator.Encrypter(), accountKey)
-	if err != nil {
-		return nil, errio.Error(err)
-	}
-
-	serviceRepoMemberRequest, err := s.client.createRepoMemberRequest(repoPath, accountKeyRequest.PublicKey)
+	serviceRepoMemberRequest, err := s.client.createRepoMemberRequest(repoPath, credentialRequest.AccountKey.PublicKey)
 	if err != nil {
 		return nil, errio.Error(err)
 	}
 
 	in := &api.CreateServiceRequest{
 		Description: description,
 		Credential:  credentialRequest,
-		AccountKey:  accountKeyRequest,
```

Comment:
I believe that error is related to this removal.

Have we changed the API server yet so that it doesn't require this to be set if the account key is supplied via the credential request?

Reply:
We haven't done this yet, so this PR should wait until we've released a version of the API server which accepts the account key either to be supplied at the root of the request (as it currently is) or embedded in the create credential request (as is done after this changeset is applied).

Reply:
@SimonBarendse The new version of the API server has been released. Could you test this again and see if you still get any errors?